### PR TITLE
Final set of rebuilds

### PR DESCRIPTION
**Pull request by ehelms**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.16
* [ ] 1.15
* [ ] 1.14

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

**Comment by ekohl** [comment]
[test]

File: rubygem-will_paginate/rubygem-will_paginate.spec [hunk 1]
```diff
@@ -17,13 +17,6 @@ Requires:      %{?scl_prefix_ruby}ruby(rubygems)
 Requires:      %{?scl_prefix_ror}rubygem(activerecord)
```
Comment (ekohl) [inline rubygem-will_paginate/rubygem-will_paginate.spec]: These dependencies are odd. The gemspec doesn't list any but they do have gemfiles in the git root for rails versions. Is this required or not?

**Comment by ekohl** [comment]
http://koji.katello.org/koji/taskinfo?taskID=66418 rubygem-net-scp failed (needs mocha)
http://koji.katello.org/koji/taskinfo?taskID=66429 rubygem-paint failed (needs rspec)

**Comment by ekohl** [comment]
Looks like those deps are only needed for tests so we can drop them if we remove the ```%check``` section.

**Comment by ehelms** [comment]
@ekohl All passing now

**Pull request merged**